boot/aether/src/boot/aether.clj

```diff
@@ -68,7 +68,7 @@
 
 (defn transfer-listener
   [info]
-  (util/dbug "Aether: %s\n" (with-out-str (pprint/pprint info)))
+  (util/trace* "Aether: %s\n" (with-out-str (pprint/pprint info)))
```

Comment:
Yep, macro version is definitively a good call here, else the pprint is called for the info even if dbug prints are disabled.